src/app/user/latest-collective/components/CreatorRowComponent.tsx

```diff
@@ -0,0 +1,34 @@
+import { CategoryColumn } from '@/app/proposals/components/table-columns/CategoryColumn'
+import { CopyButton } from '@/components/CopyButton'
+import { SmallLineSeparator } from '@/components/Separators/SmallLineSeparator'
+import { Tooltip } from '@/components/Tooltip'
+import { Paragraph, Span } from '@/components/TypographyNew'
+import { cn, shortAddress } from '@/lib/utils'
+import { ProposalCategory } from '@/shared/types'
+import { Moment } from 'moment'
+import { ClassNameValue } from 'tailwind-merge'
+import { Address } from 'viem'
+
+interface CreatorRowComponentProps {
+  proposer: Address
+  Starts: Moment
+  category: ProposalCategory
+  className?: ClassNameValue
+}
+
```

Comment:
I got confused by the name. If I read at the code I know it's for proposals. Could we be a bit more specific here?